wormchain/syncing.md

```diff
@@ -14,7 +14,7 @@ git clone https://github.com/wormhole-foundation/wormhole
 
 # checkout latest release (v2.23.0 at time of writing)
```

Comment:
can you please update this as well to be:

> checkout latest release (v2.24.2-wormchaind at time of writing)

Reply:
sure @nik-suri, updated the PR